components/radio-button/script.js

```diff
@@ -26,9 +26,11 @@ class GamefaceRadioGroup extends HTMLElement {
 
     // eslint-disable-next-line require-jsdoc
     get value() {
-        if (this.disabled) return;
-        if (this.previouslyCheckedElement && this.previouslyCheckedElement.disabled) return;
+        if (this.disabled) return null;

```

Comment:
Is there a rule that requires you to return null? If there is, why isn't it changed in other files?

Reply:
The error that I receive is Expected to return a value in getter 'value'. So I decided I had to return null or undefined.

Reply:
As for why it is not broken in other files - I guess we don't simply return in other getters :D